version/1/4/de/code_of_conduct.md

```diff
@@ -0,0 +1,43 @@
+# Vereinbarung über Verhaltenskodex für Mitwirkende
+
+## Unsere Verpflichtung
+
+Im Interesse der Förderung eines offenen und einladenden Umfeldes wollen wir uns als Teilnehmer und Verantwortliche unseres Projektes verpflichten die Teilnahme an dem Projekt und unserer Gemeinschaft zu einer belästigungsfreien Erfahrung zu machen – unabhängig von Alter, Körpergröße, Behinderung, ethnischer Zuordnung, geschlechtlicher Identität und Ausdruck, Erfahrungsstufe, Nationalität, persönlicher Erscheinung, Rasse, Religion oder sexueller Identität oder Orientierung
+
+## Unsere Standards
+
+Beispiele für Verhaltensweisen, welche dazu beitragen ein positives Umfeld zu erzeugen beinhalten
+
+* Die Verwendung von einladenden und einbindenden Formulierungen
+* Abweichenden Ansichten und Erfahrungen wird Respekt entgegengebracht
+* Konstruktive Kritik wird höflich entgegengenommen
+* Fokussierung auf das, was das Beste ist für die Gemeinschaft
+* Verständnis zeigen gegenüber anderen Mitgliedern der Gemeinschaft
+
+Beispiele für nicht akzeptables Verhalten beinhalten:
+
+* Die Verwendung sexualisierter Sprache, Bilder oder Symbolik sowie unerwünschte Versuche sexueller Anbahnung
+* Nutzung der Anonymität des Internets für unbilliges Verhalten, beleidigende / abwertende Kommentare, persönliche oder politische Angriffe
+* Öffentliche oder private Belästigungen
+* Das Veröffentlichen von privaten Informationen Anderer, wie zum Beispiel physische oder elektronische Adressen, ohne deren ausdrückliche Erlaubnis
+* Anderes Verhalten, welches begründet als unangemessen betrachtet werden kann in einem professionellen Umfeld
+
+## Unsere Verantwortlichkeiten
+
+Die Projektverantwortlichen sind verantwortlich dafür die Standards für ein akzeptables Benehmen klarzustellen und es wird von ihnen erwartete, dass sie passende und faire berichtigende Maßnahmen ergreifen als Reaktion auf jegliches Vorkommen von inakzeptablen Verhalten.
+
+Die Projektverantwortlichen haben das Recht und die Verantwortung, Kommentare, Commits, Code, Wiki-Bearbeitungen, Support-Tickets und andere Beiträge, die nicht mit diesem Verhaltenskodex vereinbar sind, zu entfernen, zu bearbeiten oder abzulehnen, und jene Mitwirkende für Verhaltensweisen, die sie für unangemessen, bedrohend, beleidigend oder verletzend halten, zeitweilig oder dauerhaft zu sperren.
+
+## Geltungsbereich
+
+Dieser Verhaltenskodex gilt sowohl innerhalb des Projektbereichs als auch in öffentlichen Bereichen, wenn eine Person das Projekt oder seine Gemeinschaft repräsentiert. Beispiele für die Repräsentation eines Projektes oder der Gemeinschaft beinhalten die Verwendung einer offiziellen Projekt-E-Mail-Adresse, das Versenden von Nachrichten über einen öffentlichen Soziale-Medien-Account oder das Handeln als Repräsentant während einer Online- oder Offline-Veranstaltung. Das Thema "Repräsentation des Projektes" kann durch die Projektenverantwortlichen weiter ausformuliert und klargestellt werden.
```

Comment:
I'd leave "Social-Media" in english because it is commonly used, but that's a matter of taste